pkg/repo/repo.go

```diff
@@ -97,6 +108,43 @@ func CreateRepo(ctx context.Context, files map[string][]byte) (tuf.LocalStore, s
 	return local, dir, nil
 }
 
+// CreateRepo creates and initializes a Tuf repo for Sigstore by adding
+// keys to bytes. keys are typically for a basic setup like:
+// "fulcio_v1.crt.pem" - Fulcio root cert in PEM format
+// "ctfe.pub" - CTLog public key in PEM format
+// "rekor.pub" - Rekor public key in PEM format
+// but additional keys can be added here.
+//
+// This will also deduce the Usage for the keys based off the filename:
+// if the filename contains:
+//   - `fulcio` = it will get Usage set to `Fulcio`
+//   - `ctfe` = it will get Usage set to `CTFE`
+//   - Anything else will get set to `Rekor`
+func CreateRepo(ctx context.Context, files map[string][]byte) (tuf.LocalStore, string, error) {
+	targets := make([]TargetWithMetadata, 0, len(files))
+	for name, bytes := range files {
+		usage := ""
+		if strings.Contains(name, "fulcio") {
+			usage = "Fulcio"
+		} else if strings.Contains(name, "ctfe") {
+			usage = "CTFE"
+		} else {
+			usage = "Rekor"
```

Comment:
I'd change this to 

```
} else if strings.Contains(name, "rekor")
     usage = "Rekor"
} else {
     usage = "Unknown"
}
```

To match https://github.com/sigstore/root-signing/blob/7187cb661647a8420c1d2ebb354bc2b7b8cade37/ceremony/2022-10-18/repository/targets.json#L17